server/src/main/antlr4/io/druid/sql/antlr4/DruidSQL.g4

```diff
@@ -1,343 +0,0 @@
-grammar DruidSQL;
```

Comment:
generated java files were failing to compile because of the constructor change. I decided to remove it instead of fixing as this is not used.